windfile.py

```diff
@@ -0,0 +1,122 @@
+import numpy as np
+import math
+import elev
+import datetime
+import bisect
+import time
+#from scipy import interpolate
+
+class WindFile:
+    def __init__(self, path):
+        self.data_npz = np.load("downtest2/" + path, "r")
+        self.data = self.data_npz['data']
+        self.time = self.data_npz['timestamp']
+        self.levels = self.data_npz['levels']
+        self.interval = self.data_npz['interval']
+        # TODO calculate resolution
+
+    def get(self, lat, lon, altitude, time):
+        if lat < -90 or lat > 90:
+            raise Exception(f"Latitude {lat} out of bounds")
+        if lon < -180 or lon > 360:
+            raise Exception(f"Longitude {lon} out of bounds")
+        
+        # TODO wrap longitude
+
+        indices = self.get_indices(lat, lon, altitude, time)
+        
+        u, v = self.interpolate(*indices)
+        
+        return u, v
+
+    def get_indices(self, lat, lon, alt, time):
+        lat = 90 - lat
+        lon = lon % 360
+
+        if isinstance(time, datetime.date):
+            time = int(datetime.timestamp(time))
+        
+        time = (time - self.time)/self.interval
+        if time < 0 or time > 65: # should this be hard coded or like len(self.data[][])?
+            raise Exception(f"Time {time} out of bounds")
```

Comment:
`self.data.shape[-2]`. Also, the time out of bounds should ideally be thrown as whatever was passed in (similar to the others)